contracts/views/CurveView.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "../actions/curve/helpers/CurveHelper.sol";
+import "../interfaces/curve/ILiquidityGauge.sol";
+
+contract CurveView is CurveHelper {
```

Comment:
Talk to frontend if they need more info here like rewards amounts, how long is crv locked etc...

Reply:
Will continue this in separate branch all good for now